flow/benchmarks/baselines/bottleneck1.py

```diff
@@ -39,84 +30,58 @@ def bottleneck1_baseline(num_runs, render=True):
         SumoExperiment
             class needed to run simulations
     """
+    exp_tag = flow_params["exp_tag"]
+    sumo_params = flow_params["sumo"]
+    env_params = flow_params["env"]
+    net_params = flow_params["net"]
+    initial_config = flow_params.get("initial", InitialConfig())
+    traffic_lights = flow_params.get("tls", TrafficLights())
+
+    # remove autonomous vehicles
     vehicles = Vehicles()
     vehicles.add(veh_id="human",
                  speed_mode=9,
                  routing_controller=(ContinuousRouter, {}),
                  lane_change_mode=1621,
                  num_vehicles=1 * SCALING)
 
-    controlled_segments = [("1", 1, False), ("2", 2, True), ("3", 2, True),
-                           ("4", 2, True), ("5", 1, False)]
-    num_observed_segments = [("1", 1), ("2", 3), ("3", 3),
-                             ("4", 3), ("5", 1)]
-    additional_env_params = {
-        "target_velocity": 40,
-        "disable_tb": True,
-        "disable_ramp_metering": True,
-        "controlled_segments": controlled_segments,
-        "symmetric": False,
-        "observed_segments": num_observed_segments,
-        "reset_inflow": False,
-        "lane_change_duration": 5,
-        "max_accel": 3,
-        "max_decel": 3,
-        "inflow_range": [1000, 2000]
-    }
-
-    # flow rate
+    # modify the inflows to only include human vehicles
```

Comment:
I think this is confusing because you're not "modifying" you're just inserting a human inflow